MIGRATION.md

```diff
@@ -2,6 +2,9 @@
 
 When new releases include breaking changes or deprecations, this document describes how to migrate.
 
+## Migrating to 0.13.0
+Jobs, ops, and graphs have replaced pipelines, solids, modes, and presets ans the stable core of the
```

Comment:
Nitpick: add surrounding newlines to match the spacing of other sections.

Also, "ans" -> "as".